eodatasets3/scripts/tostac.py

```diff
@@ -82,5 +228,26 @@ def json_fallback(o):
     )
 
 
+def validate_stac(item_doc: Dict):
+    # Validates STAC content against schema of STAC item and STAC extensions
+    stac_content = json.loads(json.dumps(item_doc, indent=4, default=json_fallback))
```

Comment:
Loads and dumps, why?

Reply:
Using dumps to JSON serialisation of objects like `UUID` to string and loads to convert back to Python objects

Reply:
I do the same in some tests for normalisation before comparing things. The indent could be removed though.